packages/vuetify/src/components/VOtpInput/VOtpInput.ts

```diff
@@ -0,0 +1,343 @@
+    value (val) {
+      this.lazyValue = val
+    },
+  },
+
+  created () {
+    /* istanbul ignore next */
+    if (this.$attrs.hasOwnProperty('browser-autocomplete')) {
+      breaking('browser-autocomplete', 'autocomplete', this)
+    }
+
+    this.otp = this.internalValue?.split('') || []
+  },
+
+  mounted () {
+    requestAnimationFrame(() => (this.isBooted = true))
+  },
+
+  methods: {
+    /** @public */
+    focus (e: Event, otpIdx: number) {
+      this.onFocus(e, otpIdx || 0)
+    },
+    genInputSlot (otpIdx: number) {
+      return this.$createElement('div', this.setBackgroundColor(this.backgroundColor, {
+        staticClass: 'v-input__slot',
+        style: { height: convertToUnit(this.height) },
+        on: {
+          click: () => this.onClick(otpIdx),
+          mousedown: (e: Event) => this.onMouseDown(e, otpIdx),
+          mouseup: (e: Event) => this.onMouseUp(e, otpIdx),
+        },
+        ref: `input-slot-${otpIdx}`,
+      }), [this.genDefaultSlot(otpIdx)])
+    },
+    genControl (otpIdx: number) {
+      return this.$createElement('div', {
+        staticClass: 'v-input__control',
+      }, [
+        this.genInputSlot(otpIdx),
+      ])
+    },
+    genDefaultSlot (otpIdx: number) {
+      return [
+        this.genFieldset(),
+        this.genTextFieldSlot(otpIdx),
+      ]
+    },
+    genCol (otpIdx: number) {
+      const node = this.$createElement('div', this.setTextColor(this.validationState, {
+        staticClass: 'v-input',
+        class: this.classes,
+      }), [this.genControl(otpIdx)])
+
+      return this.$createElement('div', {
+        staticClass: 'col-input',
+      }, [
+        node,
+      ])
+    },
+    genContent () {
+      const cols = [...Array(this.length).keys()].map(x => {
+        return this.genCol(x)
+      })
+
+      return [this.$createElement('div', {
+        staticClass: 'row-container',
+      }, cols)]
+    },
+    genFieldset () {
+      return this.$createElement('fieldset', {
+        attrs: {
+          'aria-hidden': true,
+        },
+      }, [this.genLegend()])
+    },
+    genLegend () {
+      const span = this.$createElement('span', {
+        domProps: { innerHTML: '&#8203;' },
+      })
+
+      return this.$createElement('legend', {
+        style: {
+          width: '0px',
+        },
+      }, [span])
+    },
+    genInput (otpIdx: number) {
+      const listeners = Object.assign({}, this.listeners$)
+      delete listeners.change // Change should not be bound externally
+
+      return this.$createElement('input', {
+        style: {},
+        domProps: {
+          value: this.otp[otpIdx],
+          min: this.type === 'number' ? 0 : null,
+        },
+        attrs: {
+          ...this.attrs$,
+          disabled: this.isDisabled,
+          type: this.type,
+          id: `${this.computedId}--${otpIdx}`,
+          class: `otp-field-box--${otpIdx}`,
+          maxlength: 1,
+        },
+        on: Object.assign(listeners, {
```

Comment:
The finish listener ends up in the DOM somehow:
![image](https://user-images.githubusercontent.com/16421948/111459972-822c6800-876f-11eb-8c9b-a95027d4f476.png)

Reply:
I don't know how you've got this one happening.

![image](https://user-images.githubusercontent.com/5682899/113522711-92d04100-95ab-11eb-9fdc-a28521214a3f.png)

Reply:
It was from your playground, it's still using `:on-finish` instead of `@finish`

Reply:
You're right man - just the playground was a bit outdated :|